concourse/model/resources.py

```diff
@@ -228,9 +231,9 @@ def __init__(
 
         # todo: handle "qualifier"
         if is_pull_request:
-            type_name = 'pull-request'
+            type_name = ResourceType.PULL_REQUEST.value
         else:
-            type_name = 'git'
+            type_name = ResourceType.GIT.value
 
```

Comment:
Could pbly also be replaced with the type and updating the ResourceIdentifier. Not sure, and either way I'm fine with doing this at a later point.